### PR TITLE
Avator/alerts

### PR DESCRIPTION
**Pull request by developer-anand**
Hi friends I have made avatar and made some changes please approve.

### CODE REVIEW

File: common.css [hunk 1]
```diff
@@ -8,17 +8,20 @@
   box-sizing: border-box;
 }
 
+.container-flex {
+  display: flex;
+}
+
 body {
   /* font-size: 1.6rem; */
```
Comment (zoef-yt) [inline common.css]: remove commented code

File: common.css [hunk 2]
```diff
@@ -51,3 +54,25 @@ a {
   text-decoration: none;
   color: inherit;
 }
+
+/* Box sizing rules
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
```
Comment (zoef-yt) [inline common.css]: remove commented code.

File: components/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
+@import "/common.css";
+
+/* .avatar-container {
+  box-sizing: border-box;
+  flex-grow: 1;
+  margin-left: 5rem;
+} */
```
Comment (zoef-yt) [inline components/avatar/avatar.css]: remove commented code.

File: components/avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+.avatar-lg {
+  max-height: 5rem;
+  max-width: 5rem;
+}
+
+.avatar-badge {
+  margin-top: 1.5rem;
+}
```
Comment (zoef-yt) [inline components/avatar/avatar.css:62]: add spacing

File: components/badge/badge.css [hunk 5]
```diff
@@ -0,0 +1,27 @@
+.badge-title {
+  font-size: 3rem;
+  font-weight: 700;
+}
+
+.badge-title-v2 {
+  margin-top: 2rem;
+  font-size: 2.6rem;
+  font-weight: 300;
+}
```
Comment (zoef-yt) [inline components/badge/badge.css:10]: add spacing.

File: main.css [hunk 6]
```diff
@@ -58,10 +52,34 @@ main {
   color: rgb(17, 16, 16);
 }
 
+.main-installation-header {
+  padding: 2rem;
+  font-size: 4rem;
+}
+.main-installation-left {
+  /* margin-top: 20rem; */
```
Comment (zoef-yt) [inline main.css]: remove commented code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]
remove commented code and add some spacing where suggested to stay consitent.

File: components/alerts/alert.css [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+.alert-content {
+  font-size: 1.5rem;
+  padding: 1rem;
+}
+.example-div {
+  padding: 1rem;
+  font-size: 2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 0.7rem;
```
Comment (iRohitGaur) [inline components/alerts/alert.css]: use px for values less than 10px

File: components/avatar/avatar.css [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+.badge-status {
+  position: absolute;
+  height: 1rem;
+  width: 1rem;
+  border-radius: 50%;
+  left: 5rem;
+  top: 2rem;
+}
+.badge-online {
```
Comment (iRohitGaur) [inline components/avatar/avatar.css:85]: stay consistent in spacing. if you are giving 1 line break between each class then give it the same way everywhere

File: common.css (hunk 1 above)
Comment (developer-anand) [reply common.css]: done

File: common.css (hunk 2 above)
Comment (developer-anand) [reply common.css]: done

File: components/avatar/avatar.css (hunk 3 above)
Comment (developer-anand) [reply components/avatar/avatar.css]: done

File: components/avatar/avatar.css (hunk 4 above)
Comment (developer-anand) [reply components/avatar/avatar.css:62]: can you tell what you exactly mean by spacing?

File: components/badge/badge.html [hunk 9]
```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>avator</title>
```
Comment (iRohitGaur) [inline components/badge/badge.html:7]: This is badge file. Title is wrong

File: components/button/button.html [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+            selected or deselected, such as adding or removing a star to an
+            item.
+          </div>
+          <div class="btn-icon">
+            <i class="fas fa-trash"></i>
+            <i class="fas fa-cart-plus"></i>
+            <i class="far fa-times-circle"></i>
+          </div>
+
+          <div class="btn-title">Floting button</div>
```
Comment (iRohitGaur) [inline components/button/button.html:211]: Floating*

File: components/avatar/avatar.css (hunk 4 above)
Comment (zoef-yt) [reply components/avatar/avatar.css:62]: after the block of code you have 1 line space someplace and someplace there is no spacing

File: components/alerts/alert.css (hunk 7 above)
Comment (developer-anand) [reply components/alerts/alert.css]: applied please check

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge with changes 👍🏻

File: components/avatar/avatar.css (hunk 4 above)
Comment (developer-anand) [reply components/avatar/avatar.css:62]: Thanks zoef added spacing

**Review by zoef-yt — APPROVED** [review]
good to merge

File: components/badge/badge.css (hunk 5 above)
Comment (developer-anand) [reply components/badge/badge.css:10]: added please check

File: index.html [hunk 11]
```diff
@@ -43,13 +48,50 @@ <h1 class="main-header">Build fast, responsive sites with ALPHA UI</h1>
...
+                  style="
+                  width: 100%;
+                  height: 20rem;
+                  border: 0;
+                  transform: scale(1);
+                  overflow: hidden;
+                  
+                "
+                sandbox="allow-scripts allow-same-origin"
+              >
```
Comment (sudipkundu999) [inline index.html:78]: in the deploy preview it shows : https://alphauidev.netlify.app
You can't access your CL in this way.
Import all the component css in one file say components.css and give the CDN link of that file.
it'll be like : https://alphauidev.netlify.app/components.css

File: components/avatar/avatar.css (hunk 8 above)
Comment (developer-anand) [reply components/avatar/avatar.css:85]: Thanks for suggestion.

File: index.html (hunk 11 above)
Comment (developer-anand) [reply index.html:78]: Thanks I'll do that

**Review by sudipkundu999 — APPROVED** [review]
Good to merge with changes

**Pull request merged**